src/components/Integrations/Shopify/Shopify.js

```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
-          <>
-            <p>Shopify conectado a: </p>
-            <ul>
-              {shops.map((shop) => (
-                <li key={shop.shopName}> {shop.shopName} </li>
-              ))}{' '}
-            </ul>
-          </>
-        ) : (
-          <p>Shopify desconectado</p>
-        )}
+        <div className="dp-integration" style={{ marginTop: '20px' }}>
+          {isLoading ? (
+            <>
+              <div className="dp-integration__block">
+                {shopifyHeader}
+                <div className="block">
+                  <p></p>
+                </div>
+              </div>
+              <Loading />
+            </>
+          ) : error ? (
+            <div className="dp-integration__block">
+              {shopifyHeader}
+              <div className="block">
+                <p>{error}</p>
+              </div>
+            </div>
+          ) : isConnected ? (
+            <>
+              <div className="dp-integration__block">
+                {shopifyHeader}
+                <div className="block dp-integration__status">
+                  <div className="status__info">
+                    <div>
+                      {/*<img src="img/integration__logo.png" alt="Shopify logo"/> this should be added as a class, and will be added later*/}
+                      <div className="status__data">
+                        <p>
+                          Ultima fecha de sincronización: <time>22/06/2019</time>
+                        </p>
+                        <ul>
+                          {shops.map((shop) => (
+                            <li key={shop.shopName}>
+                              {' '}
+                              <p>
+                                Nombre de la tienda: <strong> {shop.shopName} </strong>
+                              </p>{' '}
+                            </li>
+                          ))}
+                        </ul>
+                      </div>
+                    </div>
+                  </div>
+                  <button type="button" className="dp-button button-big primary-green">
+                    Desconectar
+                  </button>
+                </div>
+              </div>
+              <div className="dp-integration__block dp-integration--info">
+                <header className="block">
+                  <div>
+                    <h2>Listas creadas</h2>
+                    <p>
+                      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Et praesentium
+                      distinctio obcaecati.
+                    </p>
+                  </div>
+                  {/*<button className="dp-button button--has-icon"><span class="ms-icon icon-reload"></span></button>*/}
+                </header>
+                <hr />
+                <div className="block">
+                  <ul>
+                    {shops.map((shop) => (
+                      <li key={shop.list.id}>
+                        {' '}
```

Comment:
need this space? cant use in `<p>` styles?

Reply:
Yes I will try to remo this, is very weird. O_o